examples/electron/package.json

```diff
@@ -63,7 +63,7 @@
     "bundle": "theia build --mode development",
     "watch": "concurrently -n compile,bundle \"theiaext watch --preserveWatchOutput\" \"theia build --watch --mode development\"",
     "start": "theia start --plugins=local-dir:../../plugins",
-    "start:debug": "yarn start --log-level=debug",
+    "start:debug": "yarn start --log-level=debug --remote-debugging-port=9222",
```

Comment:
This doesn't like a valid Node.js CLI argument: https://nodejs.org/docs/latest-v12.x/api/cli.html

Reply:
Ah looks like it's an Electron flag ok: https://www.electronjs.org/docs/api/command-line-switches#--remote-debugging-portport

Reply:
right, this prepares the debug session so we can attach to it from the IDE